src/passes/CMakeLists.txt

```diff
@@ -11,7 +11,13 @@ string(APPEND WASM_INTRINSICS_EMBED "0x00")
 
 configure_file(WasmIntrinsics.cpp.in WasmIntrinsics.cpp @ONLY)
 
-FILE(GLOB passes_HEADERS *.h)
```

Comment:
It might be slightly nicer to keep using the glob up front, then use REMOVE_ITEM to remove selected headers in the emscripten build.

Reply:
done